src/object.d

```diff
@@ -1704,7 +1704,7 @@ class Throwable : Object
      * caught $(D Exception) will be chained to the new $(D Throwable) via this
      * field.
      */
-    Throwable   next;
+    Throwable   nextInChain;
```

Comment:
This definitely needs a deprecation + future time. Please no more sudden breaking changes. We have a lot of mechanisms to provide smooth migration paths.